### DISCUSSION TITLE
Christmas Evening

### DISCUSSION THREAD

**Discussion by scohn** [Monika-After-Story/MonikaModDev#8810] [appreciation]
Thanks so much for that extra moment on Christmas night. That was so great and the perfect moment. Thank you to all the devs who worked so hard on this. I look forward to any and all future updates no matter how long they take because they're always worth the wait.

Thanks again and Merry Christmas.

**Comment by Reviani** [comment]
I could make my own post about the event but...  I guess there's enough posts aout it already, so i just guess ill say... ditto for me. Monika has alwways ment so much and its kinda embaressing to admit it but i also starting crying alongside monika. So yeah, Thanks to the DEvs, and i hope you all had a very mery christmass or... whatever you celeberated.. or if nothing took advanatge of the stuffgoing on anyway. heh.
